main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -1100,22 +1123,24 @@ private CalculatorVariable getVariable(final char name, final List<CalculatorVar
      *
      * @param variables List of variables as they currently are.
      * @param variableNames String containing all the names for which variables are required
-     * @param range range of which variables are to be created (eg. 'A' <= ch && ch <= 'Z')
+     * @param theCase case for which variables are to be created (ie. Uppercase or Lowercase)
      * @param hintText text to be used as a hint when new variables are created
      * @return full list of variables in the appropriate order
      */
     private List<CalculatorVariable> sortVariables(final List<CalculatorVariable> variables,
                                                    final String variableNames,
-                                                   final Range range,
+                                                   final CaseCheck theCase,
                                                    final String hintText,
                                                    final TextWatcher textWatcher) {
         final List<CalculatorVariable> returnList = new ArrayList<>();
 
         final char[] sortedVariables = variableNames.toCharArray();
         Arrays.sort(sortedVariables);
 
-        for (final char ch : sortedVariables) {

```

Comment:
also here

Reply:
Reverted.

Reply:
Hmm, I still see the wrong version of the loop :thinking:

Reply:
Okay, that should get it.